arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -0,0 +1,361 @@
+{

```

Comment:
General Comment: The operations, parameters, models, modelProperties should all be well documented by providing accurate description.